frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/OptionListEditor.test.tsx

```diff
@@ -1,252 +1,101 @@
-      const textBox = getDescriptionInput(1);
-      await user.type(textBox, text);
-      await user.tab();
-
-      expect(handleComponentChange).toHaveBeenCalledTimes(1);
-      expect(handleComponentChange).toHaveBeenCalledWith(expectedArgs);
-    });
-
-    it('should show placeholder for option label when option list label is empty', () => {
-      renderOptionListEditor({
-        props: {
-          component: {
-            ...mockComponent,
-            options: [{ value: 2, label: '', description: 'test', helpText: null }],
-          },
-        },
-      });
-
-      expect(screen.getByText(textMock('general.empty_string'))).toBeInTheDocument();
-    });
-
-    it('should call handleComponentChange with correct parameters when removing chosen options', async () => {
-      const user = userEvent.setup();
-      renderOptionListEditor();
-      const expectedResult = ObjectUtils.deepCopy(mockComponent);
-      expectedResult.options = undefined;
-      expectedResult.optionsId = undefined;
-
-      await user.click(getDeleteButton());
-
-      expect(handleComponentChange).toHaveBeenCalledTimes(1);
-      expect(handleComponentChange).toHaveBeenCalledWith(expectedResult);
-    });
+  it('should render the open Dialog button', () => {
+    renderOptionListEditor();
+    expect(getOptionModalButton()).toBeInTheDocument();
   });
 
-  describe('LibraryOptionEditor', () => {
-    it('should render a spinner when there is no data', () => {
-      renderOptionListEditor({
-        queries: {
-          getOptionList: jest.fn().mockImplementation(() => Promise.resolve<OptionList>([])),
-        },
-        props: { component: componentWithOptionsId },
-      });
+  it('should render ManualOptionsEditor', async () => {
+    const user = userEvent.setup();
+    renderOptionListEditor();
 
-      expect(
-        screen.getByText(textMock('ux_editor.modal_properties_code_list_spinner_title')),
-      ).toBeInTheDocument();
-    });
-
-    it('should render an error message when getOptionList throws an error', async () => {
-      await renderOptionListEditorAndWaitForSpinnerToBeRemoved({
-        queries: {
-          getOptionList: jest.fn().mockImplementation(() => Promise.reject()),
-        },
-      });
-
-      expect(
-        screen.getByText(textMock('ux_editor.modal_properties_fetch_option_list_error_message')),
-      ).toBeInTheDocument();
-    });
-
-    it('should be able to remove binding to optionList if getOptionList throws an error', async () => {
-      const user = userEvent.setup();
-      await renderOptionListEditorAndWaitForSpinnerToBeRemoved({
-        queries: {
-          getOptionList: jest.fn().mockImplementation(() => Promise.reject()),
-        },
-      });
-
-      expect(getDeleteButton()).toBeInTheDocument();
-      await user.click(getDeleteButton());
-      expect(handleComponentChange).toHaveBeenCalledTimes(1);
-    });
+    await user.click(getOptionModalButton());
```

Comment:
Changing the function's name to `getEditButton` better matches the UI, making it easier to see what is what in the future:
```suggestion
    await user.click(getEditButton());
```